### PR TITLE
For those of us with separate web servers

### PR DESCRIPTION
**Pull request by stompsucks**
Please excuse me if I'm doing this all wrong; I am both a PowerShell and pull request novice.

This small update allows you to specify the path to the web server's root. Useful in cases where FileMaker's default server is not exposed to the internet.

### CODE REVIEW

File: GetSSL.ps1 [hunk 1]
```diff
@@ -36,6 +36,17 @@
...
 Param(
+	<#
+		Specify an alternate web root directory for HTTP-01 challenge. For use if FileMaker's web server
+		is not exposed to the internet, but another web server is. As long as that other server's
+		filesystem is accessible to the FileMaker server, anyway.
+
+		For example, -WebRoot '\\web-server\C$\inetpub\wwwroot'
+	#>
+	[Parameter(ParameterSetName='Setup')]
+	[Parameter(ParameterSetName='Renew')]
```
Comment (dansmith65) [inline GetSSL.ps1:47]: I think it should be included in `ScheduleTask` and `InstallCertificate` parameter set's as well, shouldn't it?

**Review by dansmith65 — COMMENTED** [review]
Other than the one comment I made, the code looks good to me. I don't have a server with this type of setup, so don't plan to test it myself. I assume you've tested it and confirmed it works as expected?

FYI: I'm using GitHub's review feature for the first time, so bear with me if I mess up this pull request/review.

File: GetSSL.ps1 (hunk 1 above)
Comment (stompsucks) [reply GetSSL.ps1:48]: Oops, you're right about ScheduleTask. I pushed another commit to include that. I'm not sure about InstallCertificate, though. The function doesn't appear to do any challenging.

**Comment by stompsucks** [comment]
> I assume you've tested it and confirmed it works as expected?

I have successfully requested and deployed a cert using these changes. The ScheduleTask option also works (now). The Renew option is only partially tested; it seems to work, but my cert hasn't actually expired yet.

This assumes that the user account running the script has write access to the web root directory on another computer. It's not an issue in my domain environment, but I can imagine auth issues in other contexts. That seems like something that can be left up to the user.

**Comment by dansmith65** [comment]
Sorry for the delay; I almost forgot about this pull request! Thanks for the contribution.

**Comment by stompsucks** [comment]
My pleasure. Thanks for your work on this project.

Hopefully Claris adds native LetsEncrypt support and makes this project obsolete. :)

**Pull request merged**